src/logger.ts

```diff
@@ -1,3 +1,5 @@
-import debug from "debug";
-
-export const logger = debug("web3-server-wallet");
+export interface ILogger {
+  info(...params: any): any;
+  debug(...params: any): any;
+  error(...params: any): any;
```

Comment:
lol, why do you have return value?

Reply:
For example winston has return value of type logger